src/note.ts

```diff
@@ -482,6 +483,26 @@ export abstract class Note extends Tickable {
     return false;
   }
 
+  getStem(): Stem | undefined {
+    return undefined;
+  }
+
+  getBeamCount(): number {
+    return 0;
```

Comment:
This feels like a potential silent failure. For notes that don't have stems, this should fail loudly, no?

Reply:
I mean, it's true though that e.g. a whole note has 0 beams. It might depend on why and how the getter is used. If i iterate over an array of notes that might contain whole notes, should it throw on the whole note?